docs/source/index.rst

```diff
@@ -80,31 +80,31 @@
             <h2 class="ui dividing header">In-Depth Guides</h2>
             <div class="ui four stackable cards">
 
-                <a class="ui link card" href="/Probability_Distributions.html">
+                <a class="ui link card" href="Probability_Distributions.html">
```

Comment:
the changes in this file are such that these pages show up when building the docs locally - is there a way to check that this would still work when they're deployed?